httpstan/models.py

```diff
@@ -193,119 +193,119 @@ def _build_extension_module(
-            ),
-        ]
-
-        stan_macros: List[Tuple[str, Optional[str]]] = [
-            ("BOOST_DISABLE_ASSERTS", None),
-            ("BOOST_NO_DECLTYPE", None),
-            ("BOOST_PHOENIX_NO_VARIADIC_EXPRESSION", None),
-            ("BOOST_RESULT_OF_USE_TR1", None),
-            ("STAN_THREADS", None),
-        ]
-
-        if extra_compile_args is None:
-            extra_compile_args = ["-O3", "-std=c++11"]
-            if platform.system() == "Windows":
-                # -D_hypot=hypot, MinGW fix, https://github.com/python/cpython/pull/11283
-                extra_compile_args.append("-D_hypot=hypot")
-
-        cython_include_path = [os.path.dirname(httpstan_dir)]
-        extension = setuptools.Extension(
-            module_name,
-            language="c++",
-            sources=[pyx_filepath.as_posix()],
-            define_macros=stan_macros,
-            include_dirs=include_dirs,
-            extra_compile_args=extra_compile_args,
-        )
-        build_extension = Cython.Build.Inline._get_build_extension()
-
-        def _has_fileno(stream) -> bool:
-            """Returns whether the stream object has a working fileno()
-
-            Suggests whether _redirect_stderr is likely to work.
-            """
-            try:
-                stream.fileno()
-            except (AttributeError, OSError, IOError, io.UnsupportedOperation):
-                return False
-            return True
-
-        def _redirect_stdout() -> int:
-            """Redirect stdout for subprocesses to /dev/null.
-
-            Returns
-            -------
-            orig_stderr: copy of original stderr file descriptor
-            """
-            sys.stdout.flush()
-            stdout_fileno = sys.stdout.fileno()
-            orig_stdout = os.dup(stdout_fileno)
-            devnull = os.open(os.devnull, os.O_WRONLY)
-            os.dup2(devnull, stdout_fileno)
-            os.close(devnull)
-            return orig_stdout
-
-        def _redirect_stderr_to(stream: IO[Any]) -> int:
-            """Redirect stderr for subprocesses to /dev/null.
-
-            Returns
-            -------
-            orig_stderr: copy of original stderr file descriptor
-            """
-            sys.stderr.flush()
-            stderr_fileno = sys.stderr.fileno()
-            orig_stderr = os.dup(stderr_fileno)
-            os.dup2(stream.fileno(), stderr_fileno)
-            return orig_stderr
-
-        # silence stdout and stderr for compilation, if stderr is silenceable
-        # silence stdout too as cythonizing prints a couple of lines to stdout
-        stream = tempfile.TemporaryFile(prefix="httpstan_")
-        redirect_stderr = _has_fileno(sys.stderr)
-        compiler_output = ""
-        if redirect_stderr:
-            orig_stdout = _redirect_stdout()
-            orig_stderr = _redirect_stderr_to(stream)
+    temporary_dir = tempfile.TemporaryDirectory().name:
```

Comment:
There is an extra `:` at the end of the line

Reply:
Thanks. Fixed.